### ISSUE TITLE
Update two links to Globus instructions

### ISSUE THREAD

**Issue by amyehodge** [sul-dlss/happy-heron#2972]
In the second Globus email, change the link for "instructions" in item # 1 on the list to https://docs.google.com/document/d/10b7y3yZCOfyVJ_uP4l7QHbkILQKnfdJi9hj-MpmmdIk/edit#heading=h.n1tmzgcygoqj.

In the new instructions for the Globus file upload option, change the link for "instructions" in Step 5 to https://docs.google.com/document/d/10b7y3yZCOfyVJ_uP4l7QHbkILQKnfdJi9hj-MpmmdIk/edit#heading=h.n1tmzgcygoq.

![Screen Shot 2023-01-20 at 3.32.55 PM.png](https://images.zenhubusercontent.com/5e9e06e81c23039800e23efc/43cc81c8-1f29-446f-83ad-eff6ddb75848)

When finished, please deploy to stage. Thanks!

**Comment by peetucket** [comment]
I think both of these should be the same links (noticed the second link above was missing a character and just went to the top of the page)?

**Comment by amyehodge** [comment]
Yes, they should be the same link. Urgh. Hard to get them right in the UI when you can't even get them right in the ticket. I've fixed it above.
